### PR TITLE
Add remaining transcriptions for html-for-beginners

### PR DESCRIPTION
**Pull request by likwidoxigen**
This is a transcripts only PR. Just wanted something straightforward to get my feet wet and so I added the remaining transcripts for this course and fixed the file name typo for 04-images

Remove Incorrectly Named File 04-images
Create 04-html-text-formatting transcript
Create 05-table-basics transcript
Create 06-table-nesting transcript
Create 07-Table Semantics Transcript

## What type of PR is this?
- [ ] Typo
- [ ] New course or tutorial documentation
- [ X ] Includes transcript

### CODE REVIEW

File: html-for-beginners/transcript/05-table-basics.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+[00:05:12] Let's do that now. So let's do width is equal to 100%. And we're going to save that. We're going to refresh the page. And can you see now that the table itself has a hundred percent width that goes spans, right? The way across the page, same with these table headers like that. Can you see that? There's this, this block here and then this block here what's happening is that the, uh, the browser is automatically calculating that we've got three, uh, three.
+
+[00:05:47] Uh, columns and therefore the, uh, it needs to be divided up in such a way that, um, that the balance of width are equal on each of these three, three [00:06:00] things. Now let's just remove that, for example, uh, for now, because I just wanna demonstrate something else. Let's just refresh that and bring it back to how it was.
```
Comment (pfwd) [inline html-for-beginners/transcript/05-table-basics.md]: Please put each timecode on a new line

**Review by pfwd — CHANGES_REQUESTED** [review]
Please put each timecode on a new line

**Comment by likwidoxigen** [comment]
Fixed. Also added markdown extension to 07-table-semantics.

**Review by pfwd — APPROVED** [review]
Thank you. Looks good

**Pull request merged**